### PR TITLE
feat(backend): seed events

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
 * sets the property `postType` to `Article` as default
 * seeds one active event
 * seeds one passed event
 * seeds one event started but not ended

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/src/db/seed.js [hunk 1]
```diff
@@ -596,7 +596,79 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
...
+        },
+      }),
+    ])
+
+    let passedEvent = await neode.find('Post', 'e1')
+    await passedEvent.update({ eventStart: new Date(2010, 8, 30, 10).toISOString() })
+    passedEvent = await neode.find('Post', 'e2')
+    await passedEvent.update({
+      eventStart: new Date(now.getFullYear(), now.getMonth(), now.getDate() - 3).toISOString(),
+    })
```
Comment (Tirokk) [inline backend/src/db/seed.js:669]: I do not understand, why you change the dates afterwards instead doing it in the first step?

PS: But you will have your thoughts and therefore it's no reason for me to not approve.

**Review by Tirokk — APPROVED** [review]
Year @Mogge ! 😍

Cool you write seedings.

I approve. 🚀💫💫

PS: I just have an understanding question.

File: backend/src/db/seed.js (hunk 1 above)
Comment (Mogge) [reply backend/src/db/seed.js:669]: The API does not allow events in with a start date in the past. Therefore I have to pick a date in the future and then date it back

**Review by mahula — APPROVED** [review]
Well well

**Pull request merged**